packages/react-native/android/build.gradle

```diff
@@ -24,6 +24,10 @@ android {
     buildToolsVersion safeExtGet('buildToolsVersion', '28.0.3')
     compileSdkVersion safeExtGet('compileSdkVersion', 28)
 
+    if (android.hasProperty('namespace')) {
+        namespace 'com.bugsnag.reactnative'
```

Comment:
So there are no issues with it still being defined in `AndroidManifest.xml` as well?

Reply:
It emits a warning when defined in both, but doesn't fail the build.